.simplecov

```diff
@@ -4,5 +4,5 @@ SimpleCov.start 'rails' do
   add_filter %w[/app/channels/ /app/jobs/ /bin/ /db/]
   coverage_dir 'artifacts'
   formatter SimpleCov::Formatter::RcovFormatter
-  minimum_coverage 70
+  minimum_coverage 60
```

Comment:
Defer to @anarchivist on this one, but I think 70% is already pretty low and I don't think we should be dropping it even further just to pass CI.